test/e2e/kubevirt.go

```diff
@@ -379,6 +379,14 @@ iperf3 -t 0 -c %[1]s -p %[2]d --logfile %[3]s &
 			}
 		}
 
+		checkNorthSouthEgressICMPTraffic = func(vmi *kubevirtv1.VirtualMachineInstance, address string, stage string) {
+			GinkgoHelper()
+			cmd := fmt.Sprintf("ping -c 3 -W 2 %s", address)
+			stdout, err := kubevirt.RunCommand(vmi, cmd, time.Minute)
```

Comment:
Since ping timeout is 2 seconds it do not make much sense to put  Minute here, let's go with 5 seconds just in case.

Reply:
DONE.
I also put both ipv4, ipv6 addresses, they both work with ping so no need to have different command for them.
```
ping -c 3 -W 2 fc00:f853:ccd:e793::8
  PING fc00:f853:ccd:e793::8(fc00:f853:ccd:e793::8) 56 data bytes
  64 bytes from fc00:f853:ccd:e793::8: icmp_seq=1 ttl=63 time=15.8 ms
  64 bytes from fc00:f853:ccd:e793::8: icmp_seq=2 ttl=63 time=11.5 ms
  64 bytes from fc00:f853:ccd:e793::8: icmp_seq=3 ttl=63 time=5.01 ms
  
  --- fc00:f853:ccd:e793::8 ping statistics ---
  3 packets transmitted, 3 received, 0% packet loss, time 2001ms
```